src/fileInput/gridReadPSAT.cpp

```diff
@@ -132,9 +132,10 @@ void loadPSAT (coreObject *parentObject, const std::string &filetext, const basi
     {
         if (!(bri.prefix.empty ()))
         {
+            std::string prefix_underscore = bri.prefix + '_';
             for (auto &vk : Vnames)
             {
-                vk = bri.prefix + '_' + vk;
+                vk = prefix_underscore.append(vk);
```

Comment:
I don't think this will work, it will append multiple times and make a long string instead of separate appends.
append is a modifying function you should make prefix_underscore  const then.
good idea but I don't think this will work as is.

Reply:
Reverted. There's one other spot with a unneeded string temporary (readElementFile.cpp, same pattern), I'll figure something else out for them in the next pr.